config/default.json

```diff
@@ -1,6 +1,21 @@
 {
   "electron": {
     "appName": "OpenSphere",
-    "releaseUrl": "https://github.com/ngageoint/opensphere-electron/releases"
+    "releaseUrl": "https://github.com/ngageoint/opensphere-electron/releases",
+    "webPreferences": {
+      //
```

Comment:
Note:[config](https://www.npmjs.com/package/config) strips comments from loaded files, so it's safe to include them.